### PR TITLE
update logic to dunesql

### PR DESCRIPTION
**Pull request by sharkxff**
I have read the CLA Document and I hereby sign the CLA

### CODE REVIEW

**Comment by sharkxff** [comment]
@Hosuke @jeff-dude Previously the pr was closed, re-pulled the items, and the checks all worked in this pr

File: models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {{ config(
-    alias = alias('trades'),
+    alias = alias('quotation_trades'),
+    tags=['dunesql'],
     partition_by = ['block_date'],
```
Comment (Hosuke) [inline models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql:4]: You can modify this spell to partition_by block_month

File: models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql [hunk 2]
```diff
@@ -27,16 +28,17 @@ WITH quo_evt AS (
            premiumNXM,
            scAdd,
            sumAssured,
-           '0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b' as token
+           cast('0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b' as varbinary) as token
```
Comment (Hosuke) [inline models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql:31]: `0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b` is varbinary type.
```suggestion
           0xd7c49cee7e9188cca6ad8ff264c1da2e69d4cf3b as token
```

**Review by Hosuke — CHANGES_REQUESTED** [review]

**Comment by sharkxff** [comment]
@Hosuke I have followed the comments to change, the data type of token here, because as the association condition of the join, do not change the data type will report an error:Cannot apply operator: varchar(42) = varbinary, now in the on here to do processing

File: models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql [hunk 3]
```diff
@@ -64,19 +66,19 @@ SELECT quo_evt.cid,
...
-       to_timestamp(quo_evt.expiry)                                AS evt_expiry_date,
-       TRY_CAST(date_trunc('DAY', quo_evt.evt_block_time) AS date) AS block_date
+       from_unixtime(TRY_CAST(quo_evt.expiry as double) ) AS evt_expiry_date,
+       TRY_CAST(date_trunc('DAY', quo_evt.evt_block_time) AS date) AS block_date,
+       TRY_CAST(date_trunc('month', quo_evt.evt_block_time) AS date) AS block_month
 FROM quo_evt
 INNER JOIN {{ source('ethereum','transactions') }} tx
-    ON quo_evt.evt_tx_hash = tx.hash
-    AND tx.success is TRUE
+    ON from_hex(quo_evt.evt_tx_hash) = tx.hash
```
Comment (Hosuke) [inline models/nexusmutual/ethereum/nexusmutual_ethereum_quotation_trades.sql:74]: We use `from_hex()` to convert hexstring varchar to varbinary.

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @sharkxff 

[Sample nexusmutual_ethereum_quotation_trades](https://dune.com/queries/2930595)

File: models/nexusmutual/ethereum/nexusmutual_ethereum_schema.yml [hunk 4]
```diff
@@ -49,6 +49,7 @@ models:
             - block_date
             - evt_tx_hash
             - evt_index
+            - cid
     columns:
```
Comment (Hosuke) [inline models/nexusmutual/ethereum/nexusmutual_ethereum_schema.yml:53]: By the way you can add a `check_seed` test here.

**Comment by sharkxff** [comment]
@Hosuke    I'm getting this error :backquoted identifiers are not supported; use double quotes to quote identifiers", but there is no such identifier in the nexusmutual_ethereum_schema.yml file, please help me to check where the problem is!

File: models/nexusmutual/ethereum/nexusmutual_ethereum_schema.yml [hunk 5]
```diff
@@ -49,6 +49,15 @@ models:
             - block_date
             - evt_tx_hash
             - evt_index
+            - cid
+      - check_seed_legacy:
```
Comment (Hosuke) [inline models/nexusmutual/ethereum/nexusmutual_ethereum_schema.yml:53]:
```suggestion
      - check_seed:
```

**Review by Hosuke — APPROVED** [review]

**Review by jeff-dude — APPROVED** [review]
thank you!

**Pull request merged**